ChangeLog

```diff
@@ -3016,6 +3016,8 @@ Changes
    * Clarify the documentation of mbedtls_ssl_setup.
    * Use (void) when defining functions with no parameters. Contributed by
      Joris Aerts. #678
+   * Update Win32 APIs used to remove those not permitted by the Windows Store.
+     Fix provided by Kevin Kane, Microsoft. #635
```

Comment:
Was this really done in Mbed TLS release 2.8.0?

Reply:
I think the intent was to attribute the original author. I will remove it add a new relevant section using changelog.d. 

The question is should we backport those changes to 2.8.0 ?

Reply:
Assuming you mean 2.28 :) the aim with LTSes is to keep things stable. Since these changes enables builds which currently don't work, they would only seem worth back-porting if there is very low risk of upsetting an existing, working, build and we think having development and LTS in sync (for easier backporting of other changes) is worth this risk.

I'm not convinced this risk is worth it, so I would argue against a backport: this PR enables builds that don't currently work, and we don't want new people using an LTS, we'd rather they used development.

(Also: could we have a nicer title for the PR, which summarises the WHY of these changes?)

Reply:
We're breaking the build, at least (after looking for 30s), if `-lbcrypt` is not available. That's not acceptable in an LTS.

In general, building an application with unchanged code for a new OS is desirable, so it's good to backport platform support for newer OSes. But we don't have to do it systematically, since it's technically a new feature. And we definitely must not do it if it reduces support for older platforms.